manifests/nexus.pp

```diff
@@ -65,9 +66,20 @@
 
   }.filter |$keys, $values| { $values != undef }
 
-  $artifact_url = assemble_nexus_url($url, $query_params)
-  $checksum_url = regsubst($artifact_url, "p=${packaging}", "p=${packaging}.${checksum_type}")
-
+  if $use_nexus3_urls {
+    if $classifier {
+      $c = "-${classifier}"
+    } else {
+      $c = ''
+    }
+    $artifact_url = sprintf('%s/repository/%s/%s/%s/%s/%s-%s%s.%s', $url,
```

Comment:
They can (but shouldn't). It would be very unusual to have, say, a space in one of the parameters.